lib/pry/pry_class.rb

```diff
@@ -161,6 +161,9 @@ def self.final_session_setup
   #   Pry.start(Object.new, :input => MyInput.new)
   def self.start(target=nil, options={})
     return if ENV['DISABLE_PRY']
+    if ENV['FAIL_PRY']
+      raise 'You have FAIL_PRY set to true, which results in pry calls failing'
```

Comment:
`which results in pry calls failing` → `which results in Pry calls failing` :stuck_out_tongue_winking_eye: